css/TagField.css

```diff
@@ -1,6 +1,7 @@
 .select2-container {
     position: relative;
     max-width: 416px !important;
+    width: 100%;
```

Comment:
I think this is necessary if the field is rendered inside a container that's hidden on load.